Formula/helidon.rb

```diff
@@ -0,0 +1,21 @@
+class Helidon < Formula
+  desc "Set of Java libraries for writing microservices that run on Netty"
```

Comment:
The Helidon CLI lets you easily create a Helidon project by picking from a set of archetypes. It also supports a developer loop that performs continuous compilation and application restart, so you can easily iterate over source code changes.

Reply:
HomeBrew policy: Description is too long, it should be less than 80 character and it cannot start with an article nor formula name.

Reply:
"Command line tool for Helidon application development".